astroquery/utils/tap/core.py

```diff
@@ -343,7 +345,7 @@ def launch_job(self, query, name=None, output_file=None,
             job._phase = 'COMPLETED'
         return job
 
-    def launch_job_async(self, query, name=None, output_file=None,
+    def launch_job_async(self, query, maxrec=None, name=None, output_file=None,

```

Comment:
new kwargs should come to the very end of the current ones, otherwise it introduces a backward incompatible API chanege